README.md

```diff
@@ -61,6 +61,31 @@ pub struct Rgb {
 }
 ```
 
+### Customizing single fields
```

Comment:
This is a subsection of the `Automatically Deriving Arbitrary` section so it should have one more `#`:

```suggestion
#### Customizing single fields
```

Reply:
Addressed in https://github.com/rust-fuzz/arbitrary/pull/129/commits/386de854122cbfab158bd99ffd7b7f4c46c8cfe9